server/src/test/java/io/unitycatalog/server/base/metastore/BaseMetastoreSummaryTest.java

```diff
@@ -0,0 +1,34 @@
+package io.unitycatalog.server.base.metastore;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.unitycatalog.client.ApiException;
+import io.unitycatalog.client.model.GetMetastoreSummaryResponse;
+import io.unitycatalog.server.base.BaseCRUDTest;
+import io.unitycatalog.server.base.ServerConfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public abstract class BaseMetastoreSummaryTest extends BaseCRUDTest {
+  protected MetastoreOperations metastoreOperations;
+
+  protected abstract MetastoreOperations createMetastoreOperations(ServerConfig serverConfig);
+
+  @BeforeEach
+  @Override
+  public void setUp() {
+    super.setUp();
+    metastoreOperations = createMetastoreOperations(serverConfig);
+  }
+
+  @Test
+  public void testMetastoreSummary() throws ApiException {
+    GetMetastoreSummaryResponse metastoreSummary = metastoreOperations.getMetastoreSummary();
+    String metastoreId1 = metastoreSummary.getMetastoreId();
+    assertThat(metastoreId1).isNotNull();
+    // Verify that it is stable
+    String metastoreId2 = metastoreSummary.getMetastoreId();
```

Comment:
is this the right way to test whether the summary is stable? shouldnt you be requesting for the `GetMetastoreSummaryResponse` again via the sdk? 

in fact, the true test is that even if the server is restarted with same db, you still get same metastore id. how is that tested?

Reply:
Ah, I did request for `GetMetastoreSummaryResponse` again. I think I messed this up while resolving some conflict/merging. 

Will see how we can add the test for server restart.

Reply:
fixed this and added test for server restart